aptos/token_bridge/sources/transfer_tokens.move

```diff
@@ -15,6 +17,10 @@ module token_bridge::transfer_tokens {
 
     const E_TOO_MUCH_RELAYER_FEE: u64 = 0;
 
+    struct EmitterCapabilityStore has key {
```

Comment:
This is nice! We should move it to `emitter.move` instead though, and document that there are two ways of getting an emitter capability.
1) use your signer and then it gets stored for you (advantage: easier if you have signer; downside: when calling form a contract, you need a resource account which is a ton of incurred complexity)
2) don't use a signer and you get back the capability directly (advantage: no signer needed, very easy to set up; downside: you have to manage the emitter capability directly)